app/controllers/users/two_factor_authentication_controller.rb

```diff
@@ -206,7 +206,6 @@ def handle_telephony_result(method:, default:, otp_delivery_selection_result:)
           otp_make_default_number: default,
         )
       elsif @telephony_result.error.is_a?(Telephony::OptOutError)
-        # clear message from https://github.com/18F/identity-idp/blob/7ad3feab24f6f9e0e45224d9e9be9458c0a6a648/app/controllers/users/phones_controller.rb#L40
         flash.delete(:info)
```

Comment:
If the code which was responsible for assigning the flash that we're cleaning up here no longer exists, do we need to clean up the flash at all? Were there regression specs added when this piece was originally introduced that could give us some assurance?